datahub-web-react/src/app/homeV3/context/HomePageProvider.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { useGlobalSettings } from '@app/context/GlobalSettingsContext';
+import { useUserContext } from '@app/context/useUserContext';
+import { PageTemplateProvider } from '@app/homeV3/context/PageTemplateContext';
+
+const HomePageProvider = ({ children }: { children: React.ReactNode }) => {
+    const { settings, loaded: globalSettingsLoaded } = useGlobalSettings();
+    const { user, loaded: userLoaded } = useUserContext();
+
+    const personalTemplate = user?.settings?.homePage?.pageTemplate || null;
+    const globalTemplate = settings.globalHomePageSettings?.defaultTemplate || null;
+
+    if (!userLoaded || !globalSettingsLoaded) return null;
```

Comment:
hmm i actually am not sure we want to do this at such a high level. maybe let's remove this for now. in the future we might want a better loading state